doc/sphinx/proof-engine/vernacular-commands.rst

```diff
@@ -252,6 +252,7 @@ to accessible objects.  (see Section :ref:`invocation-of-tactics`).
       .. coqtop:: in
 
          Require Import ZArith.
+         Require Import OmegaLemmas.
```

Comment:
This line shouldn't be necessary since `ZArith` exports `Omega` which exports `OmegaLemmas`. What am I missing?

Reply:
I’ve removed this change.